.travis.yml

```diff
@@ -1,25 +1,40 @@
-sudo: false
+dist: xenial
+
 language: python
+python:
+  - "3.6"
+  - "3.7"
+  - "3.8"
+
+cache: pip
+
 services:
-  - postgresql
+   - postgresql
+
+addons:
+  postgresql: "10"
```

Comment:
Is it possible to use latest Postgres/PostGIS-releases? E.g. Postgres 12.x and PostGIS 3.0.0?

Reply:
Prebuild postgres-12-postgis-3 packages are not available under xenial. Sticking with the current configuration.